packages/ui/core-components/src/lib/organisms/layout/header/Header.svelte

```diff
@@ -110,6 +110,7 @@
 
 <style>
 	.active {
-		@apply bg-gray-100;
+		--tw-bg-opacity: 1;
+		background-color: rgb(243 244 246 / var(--tw-bg-opacity));
```

Comment:
What does the active class do? afaict it's no longer used, but it's getting changed here.

Reply:
seems like it wasn't needed - I just took it out